pkgs/applications/version-management/scriv/default.nix

```diff
@@ -0,0 +1,57 @@
+{ lib
+, python3
+, fetchPypi
+, pandoc
+, git
+, scriv
+, testers
+}:
+
+python3.pkgs.buildPythonApplication rec {
+  pname = "scriv";
+  version = "1.2.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-u2HDD+pzFYpNGMKLu1eCHDCCRWg++w2Je9ReSnhWtHI=";
+  };
+
+  propagatedBuildInputs = [
```

Comment:
Can use `with python3Packages` to simplify these.

Reply:
Yeah, will change for consistency with other packages :+1: